packages/react/src/components/Tile/Tile.js

```diff
@@ -297,10 +306,12 @@ export class SelectableTile extends Component {
       value,
       name,
       title,
+      // eslint-disable-next-line no-unused-vars
```

Comment:
Instead of ignoring `no-unused-vars` here, we can safely remove `iconDescription` here?

Reply:
I tried that but it seemed to cause this warning to pop up if iconDescription is defined:
<img width="514" alt="Screen Shot 2020-01-08 at 10 34 17 AM" src="https://user-images.githubusercontent.com/17281178/71996865-8faca080-3202-11ea-8730-5b3574022c4e.png">
Seems like if folks are still using `iconDescription` the prop will be spread onto `...other` and cause the warning?

Reply:
Though this has been merged already - You are right. I don't see it's a problem, though, given it's a React warning that doesn't prevent user applications from working, and they can simply remove `iconDescription` to get rid of the warning.